ote/src/clj/ote/services/admin.clj

```diff
@@ -347,6 +352,57 @@
                   {::t-service/commercial-traffic? commercial?}
                   {::t-service/id id}))
 
+(defn- read-csv
+  "Read CSV from input stream. Guesses the separator from the first line."
+  [input]
+  (let [separator (csv-util/csv-separator input)]
+    (csv/read-csv input :separator separator)))
+
+(defn parse-exception-holidays-csv
+  [holidays-string]
+  (let [data (next (read-csv holidays-string))]
+    (map
+      (fn [exception]
```

Comment:
Minor thing: could one element symbol be called `element`, `ignore-day` or similar because `exception` leads distracts reader first to think about try-catch.